### PR TITLE
feat: implements textarea

### PR DESCRIPTION
**Pull request by lehmi11**
issue: https://github.com/ost-cas-fee-adv-23-24/design-system-component-library-football-fanatics/issues/21

### CODE REVIEW

File: components/textarea/Textarea.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  onChange: (event: ChangeEvent<HTMLTextAreaElement>) => void;
+  required?: boolean;
+  spellCheck?: boolean;
+  autoCorrect?: 'on' | 'off';
+  minLength?: number;
+  maxLength?: number;
+  disabled?: boolean;
+  rows?: number;
+  name?: string;
+  readOnly?: boolean;
```
Comment (bladicito) [inline components/textarea/Textarea.tsx:18]: what about:
autoCapitalize="off"
autoComplete="off"

I personally as user hate those values being active :stuck_out_tongue_closed_eyes:

File: stories/Textarea.stories.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    name: {
+      controls: { type: 'text' },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const InputDefault: Story = {
```
Comment (bladicito) [inline stories/Textarea.stories.ts]: Would be nice to add another story to have the state empty there. so that we can see the placeholder text.

**Review by bladicito — COMMENTED** [review]
@lehmi11 Thanks Looks good to me.

**Review by bladicito — APPROVED** [review]

File: components/textarea/Textarea.tsx (hunk 1 above)
Comment (lehmi11) [reply components/textarea/Textarea.tsx:18]: good idea. I will set those default values.

File: stories/Textarea.stories.ts (hunk 2 above)
Comment (lehmi11) [reply stories/Textarea.stories.ts]: consider it done :-)

**Pull request merged**